games-strategy/globulation2/patches/globulation2-0.9.4.4.patchset

```diff
@@ -514,5 +514,74 @@ index d0ca0ef..dd6e33b 100644
          env.Clone()
      except AttributeError:
 -- 
-2.30.2
+2.36.1
+
+
+From 0c4ee87101b8665f0178c21924c1e20191e08a6a Mon Sep 17 00:00:00 2001
+From: Victor Gamper <victor@wenzeslaus.de>
+Date: Sat, 24 Sep 2022 09:52:51 +0200
+Subject: Replace the home dir with the config dir in Haiku
+
+this moves the globulation config folder
+
+diff --git a/SConstruct b/SConstruct
+index dd6e33b..a376e51 100644
+--- a/SConstruct
++++ b/SConstruct
+@@ -10,6 +10,7 @@ import nsis
+ isWindowsPlatform = sys.platform=='win32'
+ isLinuxPlatform = sys.platform=='linux2'
+ isDarwinPlatform = sys.platform=='darwin'
++isHaikuPlatform = sys.platform=='haiku1'
+ 
+ 
+ def establish_options(env):
+@@ -162,6 +163,13 @@ def configure(env):
+         print("Could not find libGL or opengl32, or could not find GL/gl.h or OpenGL/gl.h")
+         missing.append("OpenGL")
+ 
++	#on Haiku, include libbe
++    if isHaikuPlatform:
++        env.Append(LIBS=['libbe'])
++        if not conf.CheckLib("libbe"):
++            print("Could not find libbe")
++            missing.append("libbe")
++
+     #Do checks for GLU, which is different on every system
+     if isDarwinPlatform and not server_only:
+         print("Using Apple's GLUT framework")
+diff --git a/libgag/src/FileManager.cpp b/libgag/src/FileManager.cpp
+index 760a04e..e6eec04 100644
+--- a/libgag/src/FileManager.cpp
++++ b/libgag/src/FileManager.cpp
+@@ -33,6 +33,12 @@
+ #include "BinaryStream.h"
+ #include "TextStream.h"
+ 
++#ifdef __HAIKU__
++#include <FindDirectory.h>
++#include <fs_info.h>
++#include <Path.h>
++#endif
++
+ // here we handle compile time options
+ #ifdef HAVE_CONFIG_H
+ #  include "config.h"
+@@ -68,7 +74,13 @@ namespace GAGCore
+ {
+ 	FileManager::FileManager(const char *gameName)
+ 	{
+-		#ifndef WIN32
++		#ifdef __HAIKU__
++		BPath path;
++		find_directory(B_USER_SETTINGS_DIRECTORY, &path);
++		path.Append(gameName);
++		addDir(path.Path());
++		#endif
++		#if !(defined(WIN32) || defined(__HAIKU__))
```

Comment:
```suggestion
+		#elif !defined(WIN32)
```

Reply:
I changed it to this, thank you
(I hope I implemented that correctly)